packages/apollo-server/scripts/prepare-package.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+# When we publish to npm, the published files are available in the root
```

Comment:
Huh? I thought the point of devdeps is that they aren't installed for all users?

Reply:
nope, unless you run `--production` you get all deps

Reply:
uh what?  I'm pretty sure that `--production` is just about what happens if you literally are in the apollo-server package directory and run `npm install`.  if you're only installing `apollo-server` because it's a dependency of your current project, devDeps won't be installed...

Reply:
`devDependencies` are not installed for transitive dependencies; `--production` only affects the root `package.json`.